funtests/transport.py

```diff
@@ -21,13 +21,19 @@
 else:
     from sha import new as _digest  # noqa
 
+if not hasattr(string, 'letters'):
```

Comment:
What version of Python does not have string.letters?  Kombu 3.0 supports 2.6 and up, and as far as I can tell it's present.